core/src/main/java/com/klaytn/caver/tx/account/AccountKeyPublic.java

```diff
@@ -92,11 +92,11 @@ public static AccountKeyPublic decodeFromRlp(String hexString) {
     }
 
     public String getX() {
-        return x;
+        return Numeric.toHexStringWithPrefixZeroPadded(Numeric.toBigInt(this.x),64);
```

Comment:
```suggestion
        return Numeric.toHexStringWithPrefixZeroPadded(Numeric.toBigInt(this.x), 64);
```
How about setting X or Y value in constructor? This is because zero padded only works in get method in this code.

Reply:
You're right. I forgot it. Thank you.